### PR TITLE
add howto-use description and DTO conventions

### PR DESCRIPTION
**Pull request by amedvedchuk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,18 @@ This application was generated using JHipster 7.8.1, you can find documentation
 
 This is a "microservice" application intended to be part of a microservice architecture, please refer to the [Doing microservices with JHipster][] page of the documentation for more information.
 
+## How to use this template
+1. Clone xm-ms-template to separate folder.
+2. Start project in you local XM environment and check if it is working, run tests.
+3. Remove `.git/` folder before project modification.
+4. Add project to new git repository where it will evolve and live.
+5. Find and replace `mstemplate` term everywhere in the projects.
+6. Find end remove everything related to `ExampleEntityFirst` (after checking how it is proposed to use)
```
Comment (sergeysenja1992) [inline README.md:13]: maybe need to be removed ExampleEntitySecond too

File: README.md [hunk 2]
```diff
@@ -4,6 +4,18 @@ This application was generated using JHipster 7.8.1, you can find documentation
...
+7. Do you need to use database:
+   - if YES - correct liquibase scripts in `resources/config/liquibase`
+   - if NO - remove liquibase scripts and commons `xm-commons-migration-db`
```
Comment (sergeysenja1992) [inline README.md:16]: in this case also need to remove XmDatabaseConfiguration

File: README.md (hunk 1 above)
Comment (amedvedchuk) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (amedvedchuk) [reply README.md]: fixed

**Review by sergeysenja1992 — APPROVED** [review]

**Pull request merged**